scripts/git/git-install-tools.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+
```

Comment:
Consider using "strict mode": http://redsymbol.net/articles/unofficial-bash-strict-mode/. I see you use -e below, but consider also -u, -o, and pipefail. (I don't see any actual pipe usage here, but doesn't hurt to add it now for when it sneaks in later)

Reply:
If we're going to make this change, let's do it for all the git scripts, in a separate PR?

Reply:
I opened https://trac.torproject.org/projects/tor/ticket/33603 for this change in tor, I'll also open tickets for other projects.

Reply:
SGTM